lib/Common/Core/SysInfo.cpp

```diff
@@ -254,7 +254,7 @@ AutoSystemInfo::SSE2Available() const
     return false; // TODO: xplat support
 #endif
 #else
-    #error Unsupported platform.
+    return false;
```

Comment:
@digitalinfinity if it's not Intel, IMO `SSE2Available()` should return false. What other possibilities here?